domain/src/main/java/com/architect/codes/mu8/interactor/comics/ComicUseCase.kt

```diff
@@ -0,0 +1,7 @@
+package com.architect.codes.mu8.interactor.comics
+
+import com.architect.codes.mu8.model.Comic
+
+interface ComicUseCase {
```

Comment:
Can we avoid a new file by putting the `ComicsUseCaseImpl` here below the interface?